src/cc/old/CCassetstx_v0.cpp

```diff
@@ -354,21 +354,21 @@ std::string CreateSell(int64_t txfee,int64_t askamount,uint256 assetid,int64_t p
             // if this is non-fungible tokens:
             if( !vopretNonfungible.empty() )
                 // set its evalcode
-                cpAssets->evalcodeNFT = vopretNonfungible.begin()[0];
+                cpAssets->evalcodeAdd = vopretNonfungible.begin()[0];
 
 			CPubKey unspendableAssetsPubkey = GetUnspendable(cpAssets, NULL);
-            mtx.vout.push_back(MakeTokensCC1vout(EVAL_ASSETS, cpAssets->evalcodeNFT, askamount, unspendableAssetsPubkey));
+            mtx.vout.push_back(MakeTokensCC1vout(EVAL_ASSETS, cpAssets->evalcodeAdd, askamount, unspendableAssetsPubkey));
             mtx.vout.push_back(MakeCC1vout(EVAL_ASSETS, txfee, mypk));  //marker (seems, it is not for tokenorders)
             if (inputs > askamount)
                 CCchange = (inputs - askamount);
             if (CCchange != 0)
                 // change to single-eval or non-fungible token vout (although for non-fungible token change currently is not possible)
-                mtx.vout.push_back(MakeTokensCC1vout((cpAssets->evalcodeNFT) ? cpAssets->evalcodeNFT : EVAL_TOKENS, CCchange, mypk));	
+                mtx.vout.push_back(MakeTokensCC1vout((cpAssets->evalcodeAdd) ? cpAssets->evalcodeAdd : EVAL_TOKENS, CCchange, mypk));	
 
 			std::vector<CPubKey> voutTokenPubkeys;
 			voutTokenPubkeys.push_back(unspendableAssetsPubkey);   
 
-            return FinalizeCCTx(mask, cpTokens, mtx, mypk, txfee, 
+            return FinalizeCCTx(0, cpTokens, mtx, mypk, txfee, 
```

Comment:
can `mask` (line 320) be removed? Also may be able to be removed elsewhere in this file (lines 508, 585, 682).